pkg/common/util/webhooks.go

```diff
@@ -15,6 +15,9 @@ func ValidateManagedBy(runPolicy *v1.RunPolicy, allErrs field.ErrorList) field.E
 		if len(manager) > v1.MaxManagedByLength {
 			allErrs = append(allErrs, field.TooLongMaxLength(fieldPath, manager, v1.MaxManagedByLength))
 		}
+		if manager != v1.MultiKueueController || manager != v1.KubeflowJobsController {
+			allErrs = append(allErrs, field.NotSupported(fieldPath, manager, []string{v1.MultiKueueController, v1.KubeflowJobsController}))
+		}
```

Comment:
nit: declare the list of supported values in var, and check if the value belongs to them, similarly as here: https://github.com/kubernetes/kubernetes/blob/099a88370d017dacf16e67306ebcdec8394fae83/pkg/apis/batch/validation/validation.go#L83C2-L83C46

Reply:
good one, thanks